README.md

```diff
@@ -39,14 +103,20 @@ https://github.com/todogroup/ospo101
 
 ## OSPO Examples
 
+* [Aiven](https://aiven.io/blog/open-source-program-office-at-aiven-a-year-later)
 * [Autodesk](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/autodesk.md)
 * [Capital One](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/capitalone.md)
 * [Comcast](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/comcast.md)
-* [Facebook](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/facebook.md)
+* [Meta](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/facebook.md)
 * [Microsoft](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/microsoft.md)
+* [Postman](https://dev.to/postman/how-postmans-building-their-open-source-program-office-ospo-hgf)
 * [Red Hat](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/redhat.md)
+* [RIT](https://todogroup.org/guides/casestudies/rit/)
 * [Salesforce](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/salesforce.md)
 * [SAP](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/sap.md)
+* [UC Santa Cruz](https://research.redhat.com/blog/article/building-a-university-ospo-bolstering-academic-research-through-open-source/)
 * [Uber](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/uber.md)
-* [Yahoo/Verizon Media](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/oath.md)
+* [Yahoo](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/oath.md)
```

Comment:
Seems like the link is for Oath Inc. Yahoo use case is not included in `https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies`

```suggestion
* [Oath Inc](https://github.com/todogroup/todogroup.github.io/blob/master/content/en/guides/casestudies/oath.md)
```

Reply:
Did a bit of research! Looks like Oath Inc is the umbrella name of Yahoo! & AOL (initially owned by Verizon). But that has changed since 2021.

From [Wikipedia page](https://en.wikipedia.org/wiki/Yahoo!_Inc._(2017%E2%80%93present)):

> On May 3, 2021, Verizon announced that the Verizon Media would be acquired by [Apollo Global Management](https://en.wikipedia.org/wiki/Apollo_Global_Management) for roughly $5 billion, and would simply be known as Yahoo following the closure of the deal, with Verizon retaining a minor 10% stake in the new group

Reply:
so, maybe we should keep it as yahoo 🤔 what do you think @UlisesGascon ?

Reply:
Yes! makes sense @anajsana. Thanks for the research 🙂